src/hooks/useLogDownloader/index.ts

```diff
@@ -33,7 +34,8 @@ import { getBytesAsString } from "utils/string";
 const useLogDownloader = (
   url: string,
   logType: LogTypes,
-  downloadSizeLimit: number = LOG_FILE_SIZE_LIMIT
+  downloadSizeLimit: number = LOG_FILE_SIZE_LIMIT,
+  isLoadingEvergreen: boolean = false
 ) => {
```

Comment:
May be worth making these arguments into an object, since it's a bit difficult to read when it's being called in other files